src/Sentry/Protocol/Metrics/DistributionMetric.cs

```diff
@@ -0,0 +1,35 @@
+using Sentry.Extensibility;
+using Sentry.Internal.Extensions;
+
+namespace Sentry.Protocol.Metrics;
+
+/// <summary>
+/// Distributions track a list of values over time in on which you can perform aggregations like max, min, avg.
+/// </summary>
+internal class DistributionMetric : Metric
+{
+    public DistributionMetric()
+    {
+        Value = new List<double>();
+    }
+
+    public DistributionMetric(string key, double value, MeasurementUnit? unit = null,
+        IDictionary<string, string>? tags = null, DateTime? timestamp = null)
+        : base(key, unit, tags, timestamp)
+    {
+        Value = new List<double>() { value };
+    }
+
+    public IList<double> Value { get; set; }
```

Comment:
We offer a Add method but we expose `IList` which has Add/Remove etc. Do we really want this as a public property?
Even `set` is public

Reply:
Yeah good call... I'll wrap it.